terraform-provider-constellation/docs/data-sources/attestation.md

```diff
@@ -32,17 +32,27 @@ data "constellation_attestation" "test" {
   * `gcp-sev-es`
 - `csp` (String) CSP (Cloud Service Provider) to use. (e.g. `azure`)
 See the [full list of CSPs](https://docs.edgeless.systems/constellation/overview/clouds) that Constellation supports.
+- `image` (Attributes) Constellation OS Image to use on the nodes. (see [below for nested schema](#nestedatt--image))
```

Comment:
this make it necessary to specify an output that was not required before, but seems good to make it more explicit.